src/components/ClientCard/ClientCard.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { setActiveChat } from '../../store/Chat/actions';
+import './ClientCard.css';
+
+const ClientCard = props => (
+  <li className="ClientCard" >
+    <button onClick={() => props.setActiveChat(props.chatId)}>
```

Comment:
I'm on the fence about the following, should we:
A) Turn @broneks's Button class into any Interactive (read: onClick, etc) element since eslint requires us to use interative elements for onClick stuff,
B) Remove the eslint rule for electron apps,
C) Keep things the way they are

All of which I'm cool with so theres no wrong answer, I'm just wondering if there is a more correct path, thoughts?